examples/turn-server/lt-cred-turn-rest/main.go

```diff
@@ -0,0 +1,72 @@
+// Package main implements a TURN server using
+// ephemeral credentials.
```

Comment:
Please add a short description of what's going on here to `examples/README.md`.  No need to go into details, just the basic usage like [here](https://github.com/giavac/turn/tree/master/examples#lt-creds). This can go right after the existing `lt-creds` description.